test/generators/csharp/CSharpGenerator.spec.ts

```diff
@@ -150,6 +150,36 @@ describe('CSharpGenerator', () => {
     expect(enumModel.result).toMatchSnapshot();
     expect(enumModel.dependencies).toEqual([]);
   });
+
+  test('should render enums with translated special characters', async () => {
+    const doc = {
+      $id: 'States',
+      type: 'string',
+      enum: ['test+', 'test', 'test-', 'test?!', '*test']
+    };
+    
+    generator = new CSharpGenerator({ presets: [
+      {
+        enum: {
+          self({ content }) {
+            return content;
+          },
+        }
+      }
+    ]});
```

Comment:
```suggestion

```

You don't need any custom generator here, so this is redundant 🙂 It will just [use the default generator](https://github.com/asyncapi/modelina/blob/7e278f9c6a628b8fe23125f22f3ea81bcaf7029a/test/generators/csharp/CSharpGenerator.spec.ts#L6).

Reply:
Ohh right. I forgot to check the `beforeEach` function